SDL-IREC 2019 SPEXTRO/SDL-IREC_SPEXTRO.tex

```diff
@@ -0,0 +1,304 @@
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+
+This payload will test how microgravity affects the folding of proteins. While this topic has been explored at a much larger scale and with a much greater budget, such as NASA preforming crystal spectrography of proteins folded in micro gravity on the ISS, this type of experiment has never been done at this scale. The goal of this project is to explore if protein folding in micro gravity can be done as a payload for the SDL Payload Challenge at IREC 2019. 
+
+To accomplish this, the team will construct a microfluidic spectroscopy experiment that fits within the payload restrictions of the competition. The payload itself must be safe to fly within the competition launch vehicle, provided by our chosen launch provider, and should be capable of operating successfully in the desert environment of Las Cruces, as well as safely returning to Earth intact as specified by SDL competition rules. 
+
+\section{Secondary Objective}
+\label{sec:secondary-obj}
+
+This project builds upon the experience and work done for IREC 2018. The IREC 2018 Technical Report identifies several key areas for improvement that should be pursued. Testing methodology and execution should be emphasized. Several system defects were only discovered during final system integration. Component and subsystem test plans should be formulated and executed well before the deadline to hand-off the payload to the launch team.
+
+Ease of use in manufacturing, assembly, and operation should be considered paramount. For most payload types, mass margins are sufficient enough to trade mass saving designs and machining work for quicker production times. Experience from IREC 2018 clearly identifies the need to have physical access to the internals of the payload frequently over the course of the launch campaign. Considerations should be made about ease of access and speed of final assembly and disassembly. Finally, due to the nature of the IREC competition, payloads can spend extended periods of time loaded in rockets prior to a launch opportunity (> 2 hours). The system should allow for inert storage in the launch vehicle, until the rocket is on the launch pad.
+
+Two-way Command and Data Handling (C\&DH) should be included and improvements to transmit and receive throughput should be considered. Signal interrupts and the overhead of task switching between radio modes led to less than expected data throughput. Data received by the ground station was critical in understanding payload performance, since the physical data was lost due to external recovery complications.
```

Comment:
The reason to include data transmitted back to ground during flight is great, but why two-way comms (more specifically why does the payload need to also receive)? Do you need to remotely send information or commands to the payload?

Having it on there in order to demonstrate 2way comms is fine but does add some extra complexity. It's not apparent that you need more than 1 way comms (payload-->ground)